SwiftCompilerSources/Sources/SIL/Operand.swift

```diff
@@ -48,6 +48,13 @@ public struct Operand : CustomStringConvertible, NoReflectionChildren {
   public var description: String { "operand #\(index) of \(instruction)" }
 }
 
+extension Value {
+  public typealias FilteredUses = LazyFilterSequence<UseList>
+  public var lifetimeEndingUses: FilteredUses {
```

Comment:
IMO, it would make sense to add this to `extension Sequence where Element == Operand`, so that it can be combined with other use-filters

Reply:
Yes